prolog/comment.pl

```diff
@@ -8,10 +8,8 @@
 
 dcgtrue(U,U).
```

Comment:
Doesn't look like this is used any more.

Reply:
yep removed now